src/app/components/header/header.component.ts

```diff
@@ -1,19 +1,24 @@
 import { Component, OnInit } from '@angular/core';
 import { CATEGORIES } from '@mocks/mock-categories';
+import { CartService } from '@shared/services/cart.service';
 
 @Component({
-  selector: 'app-header',
-  templateUrl: './header.component.html',
-  styleUrls: ['./header.component.scss']
+    selector: 'app-header',
+    templateUrl: './header.component.html',
+    styleUrls: ['./header.component.scss']
 })
 export class HeaderComponent implements OnInit {
 
-  categories: {}[] = CATEGORIES;
+    categories: {}[] = CATEGORIES;
 
-  constructor() {
-  }
+    constructor(private cartService: CartService) {
+    }
 
-  ngOnInit(): void {
-  }
+    ngOnInit(): void {
+    }
+
+    get totalNumberOfGoods(): number {
```

Comment:
Why do you need this getter? Can't it be done on init stage?

Reply:
Because I have a clear init and I don't need a variable declaration. Also, this value changes